### ISSUE TITLE
How to install this module?

### ISSUE THREAD

**Issue by shinesoftware** [basz/SlmLocale#66]
I am using composer.phar to install the module as described in your documentation but I get this error:

```
"slm/locale": ">=0.1.0,<1.2.0-dev",
```

Problem 1
    - The requested package slm/locale could not be found in any version, there may be a typo in the package name.

**Comment by CreativeNative** [comment]
Yep, same error here. Looks like all versions above 1.* are gone. Try "slm/locale": "dev-master"

**Comment by shinesoftware** [comment]
yes the problem change but the module is not installed at all.

    $ php composer.phar update
    Loading composer repositories with package information

This is the error shown:

    Could not fetch https://api.github.com/repos/darkmatus/roleuserbridge/git/refs/heads?per_page=100, enter your GitHub credentials to go over the API rate limit

**Comment by CreativeNative** [comment]
OK, but I think that error has nothing to do with slmlocale. Delete all other modules in your composer.json file except "slm/locale": "dev-master" and try to install or update again. If it works, than you have to search for the error in another module.

**Comment by shinesoftware** [comment]
Another error is appeared:

    Your requirements could not be resolved to an installable set of packages.

     Problem 1
         - Installation request for slm/locale dev-master -> satisfiable by slm/locale[dev-master].
         - slm/locale dev-master requires ext-intl * -> the requested PHP extension intl is missing from your system.

**Comment by CreativeNative** [comment]
I don't know what system you are running, but try on command line "php -i | grep intl". If this command doesn't find the intl extension, what I think, than install it.

sudo apt-get install php5-intl

**Comment by shinesoftware** [comment]
Intl is already installed.
![schermata 2015-01-22 alle 16 42 20](https://cloud.githubusercontent.com/assets/1553035/5858732/b3559ace-a255-11e4-9260-43e71631d1e8.png)

**Comment by CreativeNative** [comment]
Well, then I have no idea.

**Comment by shinesoftware** [comment]
me too! I don't understand why this module cannot be installed

**Comment by basz** [comment]
if `php -i | grep intl` does not give any result then intl is not active for cli.

what you show in the output is not phpinfo from cli

try : sudo php5enmod -s cli intl

**Comment by shinesoftware** [comment]
Wrong command?
    
    sudo: php5enmod: command not found

**Comment by basz** [comment]
on what system are you?

Could you make a gist at gist.github.com with the output of `php -i`

**Comment by shinesoftware** [comment]
here we go http://snipet.co.uk/PM

**Comment by basz** [comment]
Seems you need to compile php with the --enable-intl argument.

Did you compile by hand?
I would recommend 'brew' to install packages on osx.

**Comment by shinesoftware** [comment]
I have installed the Intl by a simple:

    # brew install icu4c
    # sudo pecl install intl

**Comment by shinesoftware** [comment]
I have found 2 different php installation on my MacBook Pro. 

The first one is the built-in apache webserver on Yosemite OS version and the second one has been installed when I needed to use the gettext module. I have installed a new version of php compiled with the gettext option. 

This is an OT question but ...
Is there a way to compile the Php module of the built-in version on the Yosemite Os?
